admin_guide/managing_networking.adoc

```diff
@@ -320,6 +320,14 @@ the number of local MAC addresses that the network interface can handle before
 falling back to filtering MAC addresses in software.
 ====
 
+[IMPORTANT]
+====
+Currently, the egress router is not compatible with Amazon AWS due to AWS not
+compatible with macvlan traffic.
+
+See link:https://aws.amazon.com/answers/networking/controlling-vpc-egress-traffic/[the Amazon AWS documentation for alternate methods for egress routing].
```

Comment:
No, nothing there helps OpenShift *users*, who need a solution inside OpenShift (that is aware of the difference between different pods, etc). It just describes different ways that the OpenShift developers might be able to make this feature work on AWS.